newton/tests/test_collision_pipeline.py

```
@@ -485,7 +485,13 @@ def test_shape_collision_filter_pairs(test, device, broad_phase: str):
         for i in range(n):
             s0 = int(contacts.rigid_contact_shape0.numpy()[i])
             s1 = int(contacts.rigid_contact_shape1.numpy()[i])
-            pair = (min(s0, s1), max(s0, s1))
+            t1 = model.shape_type.numpy()[s0]
+            t2 = model.shape_type.numpy()[s1]
+            if t1 > t2:
+                s0, s1 = s1, s0
+            elif t1 == t2 and s1 > s2:
```

Ruff (0.15.11)

[error] Undefined name `s2`

(F821)